python/cudf/cudf/tests/test_datetime.py

```diff
@@ -2150,6 +2150,12 @@ def test_daterange_pandas_compatibility():
     assert_eq(expected, actual)
 
 
+def test_strings_with_utc_offset_not_implemented():
+    with pytest.warns(DeprecationWarning, match="parsing timezone"):
```

Comment:
And this one is pandas in the pd.to_datetime call?

Reply:
This one is in cupy. I'll add a comment referencing this